src/hotspot/share/opto/regmask.hpp

```diff
@@ -105,6 +105,7 @@ class RegMask {
          SlotsPerVecX = 4,
          SlotsPerVecY = 8,
          SlotsPerVecZ = 16,
+         SlotsPerRegVMask = X86_ONLY(2) NOT_X86(1)
```

Comment:
I am fine with current code. Since we don't have other SlotsPerRegVMask* yet, we will treat it as variable-sized slots for SVE, which is 1.

Reply:
Ok, maybe set `SlotsPerRegVMask = 2` unconditionally then?

Reply:
Since SVE predicate reg size is 1 ~ 8 slots, I think 1 would be better here, which can save one mask bit and make bit mask handling easier (e.g. allocation on stack).